UI/FleetWnd.cpp

```diff
@@ -1837,50 +1837,33 @@ class FleetsListBox : public CUIListBox {
         if (ClientPlayerIsModerator())
             return; // todo: handle moderator actions for this...
 
-        if (!dropped_fleets.empty()) {
-            //DebugLogger() << " ... processing dropped " << dropped_fleets.size() << " fleets";
-            // dropping fleets.  get each ships of all source fleets and transfer to the target fleet
-
-            for (auto& dropped_fleet : dropped_fleets) {
-                if (!dropped_fleet) {
-                    ErrorLogger() << "FleetsListBox::AcceptDrops  unable to get dropped fleet?";
-                    continue;
-                }
-
-                // get fleet's ships in a vector, as this is what FleetTransferOrder takes
-                const std::set<int>& ship_ids_set = dropped_fleet->ShipIDs();
-                const std::vector<int> ship_ids_vec(ship_ids_set.begin(), ship_ids_set.end());
+        // Collected ship ids for the transfer.
+        std::vector<int> ship_ids;
```

Comment:
might be worth reserving space enough to fit contents of size `dropped_fleets.size() + dropped_ships.size()`